src/test/java/nb/kafka/operator/util/FieldUtil.java

```diff
@@ -0,0 +1,21 @@
+package nb.kafka.operator.util;
+
+import java.lang.reflect.Field;
+
+public final class FieldUtil {
```

Comment:
I couldn't find where is this used?

Reply:
Ah yes, we were planning to write more tests but ran out of time. This helper will be used later ; I'll remove it from this PR